README.md

```diff
@@ -3,7 +3,7 @@ void-elements
 
 ### Array of "void elements" defined by the HTML specification
 
-Exports an Array of "void element" node names as defined by the HTML spec.
+Exports a Map/Object of "void element" node names as defined by the HTML spec.
```

Comment:
I don't think it should be `Map` because that means something very specific in ES6.  Object seems fine.